README.md

```diff
@@ -44,21 +44,21 @@ In this workflow we build a new image with our playbook, setup secrets (private
 
 Relative path to playbook file relative to project source. This is mounted in the container at **/opt/app-root/src/PLAYBOOK_FILE**.
 
-**`INVENTORY_FILE`** (optional)
+**`INVENTORY_FILE`** (optional, but an inventory is required)
```

Comment:
> but an inventory is required

This isn't clear. This is a mutually exclusive option: one of (INVENTORY_FILE | INVENTORY_URL | DYNAMIC_SCRIPT_URL). We could omit it here and add a paragraph above discussing the inventory hierarchy, how it works.

Or "(optional, but an inventory file, URL or script must be specified)"